### PR TITLE
Compute x_tons based on edge coverage

### PR DESCRIPTION
**Pull request by dliyanage**

### CODE REVIEW

**Comment by mirzamomen** [comment]
It looks good to me. Aren't you getting reasonable results?

File: afl-fuzz.c [hunk 1]
```diff
@@ -935,6 +951,8 @@ static inline u8 has_new_bits(u8* virgin_map) {
 
 #else
 
+
+
```
Comment (mboehme) [inline afl-fuzz.c:955]: Whitespace nit (revert).

File: afl-fuzz.c [hunk 2]
```diff
@@ -944,11 +962,12 @@ static inline u8 has_new_bits(u8* virgin_map) {
       }
 
       *virgin &= ~*current;
-
+
```
Comment (mboehme) [inline afl-fuzz.c:965]: Whitespace nit (revert).

File: afl-fuzz.c [hunk 3]
```diff
@@ -3150,7 +3172,6 @@ static u8 save_if_interesting(char** argv, void* mem, u32 len, u8 fault) {
 
   }
 
-
```
Comment (mboehme) [inline afl-fuzz.c:3153]: Whitespace nit (revert).

File: afl-fuzz.c [hunk 4]
```diff
@@ -3292,7 +3313,6 @@ static u8 save_if_interesting(char** argv, void* mem, u32 len, u8 fault) {
         simplify_trace((u32*)trace_bits);
 #endif /* ^__x86_64__ */
 
-        if (!has_new_bits(virgin_crash)) return keeping;
```
Comment (mboehme) [inline afl-fuzz.c:3295]: Why remove?

**Review by mboehme — APPROVED** [review]
Thanks Danushka! Few whitespace nits. Otherwise LGTM.

File: afl-fuzz.c (hunk 4 above)
Comment (dliyanage) [reply afl-fuzz.c:3295]: Oopz. Let me fix it.

**Pull request merged**